libdevcore/CommonJS.cpp

```diff
@@ -52,12 +58,21 @@ bytes unpadded(bytes _b)
 	return _b;
 }
 
-std::string unpadLeft(std::string _b)
+bytes unpadLeft(bytes _b)
 {
-	auto p = _b.find_first_not_of('0');
-	if (p == std::string::npos)
-		return "0";
-	return _b.substr(p, _b.length() - 1);
+	int i = 0;
+	if (_b.size() == 0)
+		return _b;
+
+	while (true)
```

Comment:
Will cause exception is if there is no zero byte in the vector.
You can use 
while(i < _b.size() && _b[i] == 0)
   i++;